library/src/main/java/com/getstream/sdk/chat/viewmodel/ChannelViewModel.java

```diff
@@ -979,9 +979,9 @@ public void sendMessage(Message message, MessageCallback callback) {
         }
 
         // TODO: I'm reading this code and i don't get what it does, needs some clarification (Thierry)
-        if (client().isConnected()) {
-            channel.getChannelState().setReadDateOfChannelLastMessage(client().getUser(), message.getCreatedAt());
-        }
+//        if (client().isConnected()) {
```

Comment:
Just delete if it's not really needed. Or leave a comment about why it's needed to be here commented.